### PR TITLE
Remove app manifest file

### PR DESCRIPTION
**Pull request by NextTurn**
This file is not used at all. Created #457 as a replacement.

### CODE REVIEW

**Comment by NextTurn** [comment]
Fixes #210

File: src/Core/ServiceWrapper/app.manifest [hunk 1]
```diff
@@ -0,0 +1,10 @@
+﻿<assembly manifestVersion="1.0" xmlns="urn:schemas-microsoft-com:asm.v1">
+  <assemblyIdentity version="1.0.0.0" name="WinSW" type="win32" />
+  <trustInfo xmlns="urn:schemas-microsoft-com:asm.v2">
+    <security>
+      <requestedPrivileges xmlns="urn:schemas-microsoft-com:asm.v3">
+        <requestedExecutionLevel level="requireAdministrator" uiAccess="false" />
```
Comment (oleg-nenashev) [inline src/Core/ServiceWrapper/app.manifest]: uiAccess is `false` for the service mode, but it is an option for the CLI mode

Comment (NextTurn) [reply src/Core/ServiceWrapper/app.manifest]: IIUC we'll need a digital signature to enable `uiAccess`.

Comment (NextTurn) [reply src/Core/ServiceWrapper/app.manifest]: What does `uiAccess` do?

**Comment by NextTurn** [comment]
NOTE: It's also possible to defer privilege requests until the actual operations, i.e. we don't trigger UAC for `help` or `version`, or if the config file contains invalid data. This will make the codes more complicated but may be more user-friendly.

File: src/Core/ServiceWrapper/app.manifest (hunk 1 above)
Comment (NextTurn) [reply src/Core/ServiceWrapper/app.manifest]: Could you please explain why we need UIAccess? I've got a different direction for this.

**Comment by oleg-nenashev** [comment]
If it is not a big deal to exclude some commands from UAC, +100 for doing so. We may want to add more commands which do not require admin privileges later, so better to keep UX now

**Pull request merged**